src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+    private double hypotenuse;
+
+    public RightTriangle(Color color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+        this.hypotenuse = Math.sqrt(base * base + height * height);

```

Comment:
Instead of calculating the hypotenuse in the constructor, you could create a separate method to calculate it when needed. This will make the code cleaner and avoid storing unnecessary data.